File: School/Python/Assignment/Bubblesort/bubblesort.js
function bubblesort(arr){
    var swapped;
    do {
        swapped = false;
        for (var i = 0; i < arr.length; i++) {
        if (arr[i] > arr[i+1]) {
            var temp = arr[i];
            arr[i] = arr[i+1];
            arr[i+1] = temp;
            swapped = true;
        }
        }
    } while (swapped);
    console.log(arr);
}

var arr = [];
for (var i = 0; i < 10; i++) {
    arr.push(Math.floor(Math.random() * 100));
}

bubblesort(arr);

File: School/Python/Assignment/Bubblesort/bubblesort.py
import random
def bubblesort(array):
    for i in range(len(array)):
        for j in range(len(array)-1):
            if array[j] > array[j+1]:
                array[j], array[j+1] = array[j+1], array[j]
    return array

array = [random.randint(0,100) for i in range(10)]
print(bubblesort(array))